src/window-load-settings-helpers.coffee

```diff
@@ -5,15 +5,6 @@ windowLoadSettings = null
 
 exports.getWindowLoadSettings = ->
   windowLoadSettings ?= JSON.parse(window.decodeURIComponent(window.location.hash.substr(1)))
-  clone = _.deepClone(windowLoadSettings)
-
-  # The windowLoadSettings.windowState could be large, request it only when needed.
-  clone.__defineGetter__ 'windowState', ->
-    remote.getCurrentWindow().loadSettings.windowState
-  clone.__defineSetter__ 'windowState', (value) ->
-    remote.getCurrentWindow().loadSettings.windowState = value
-
-  clone
```

Comment:
The window state used to be considered part of the 'load settings', even though it actually used a completely different IPC mechanism under the hood. AFAIK, the reason that it ever had anything to do with the load settings is that for windows w/ no project paths (and hence no key in our on-disk storage system), we needed a place to store the window state when refreshing the window.

There are now separate APIs on the `ApplicationDelegate` for reading and writing this in-process, in-memory state: `getTemporaryWindowState` and `setTemporaryWindowState`. They are async.